charts/region/pipelines/templates/app-pipeline/rbac/pipeline-sa.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ServiceAccount
+imagePullSecrets:
+- name: image-registry-credentials
+metadata:
+  name: pipeline
+  annotations:
+    argocd.argoproj.io/compare-options: IgnoreExtraneous
+    argocd.argoproj.io/syncOptions: ServerSideApply=true
```

Comment:
Do we want `argocd.argoproj.io/sync-options: ServerSideApply=true, Validate=false` here? Otherwise if that is not the whole manifest, the resource will always be yellow in argo. If the above is the whole manifest, this probably does not apply

Reply:
this is the entire manifest - if i were only using a subset of the manifest (like a deployment or whatever) then i would need the validate=false...